libraries/Microsoft.Bot.Builder.Dialogs/DialogTurnStatus.cs

```diff
@@ -26,5 +26,10 @@ public enum DialogTurnStatus
         /// dialogs to the current context are on the dialog stack.
         /// </summary>
         Cancelled,
+
+        /// <summary>
+        /// Current dialog completed successfully, but turn should end.
+        /// </summary>
+        CompleteAndWait,
```

Comment:
>CompleteAndWait [](start = 8, length = 15)

what's the scenario for this?  Not opposed to adding new statuses just want to understand the scenario better.

Reply:
I see what you're doing. You're pausing between steps. I'd still like to better understand the scenario.

---
In reply to: [343888615](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r343888615) [](ancestors = 343888615)

Reply:
I see... you're using this in Ask. That seems reasonable.

---
In reply to: [343894758](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r343894758) [](ancestors = 343894758,343888615)